### ISSUE TITLE
Blog section needs work

### ISSUE THREAD

**Issue by mjashanks** [aletheia-foundation/aletheia-foundation.github.io#13]
http://aletheia-foundation.io/blog lists all blog posts. Currently it works, but is a bit basic.

Lets make it better. A few basic improvements:
1. Style more fitting with the rest of the site
2. Include a bit more detail.. e.g. first paragraph, with a "read more.." (not sure if this is possible with Jekyll)
3. Pagination (there is only one post at the moment, so this is maybe not a pressing issue

Any other ideas welcome, keeping in mind that a future site redesign may happen soon.

**Comment by r3versi** [comment]
**2.** Jekyll has a simple builtin "post preview": post.excerpt prints out the first paragraph of the post.

**Comment by AshBardhan** [comment]
Are you expecting this blog to be responsive?

**Comment by mjashanks** [comment]
@AshBardhan  Yes, the whole site should be responsive. I have done a bit of work on this, which may be enough for now. Unless anyone has any more comments on it.

**Comment by AshBardhan** [comment]
@mjashanks OK. However, I can do some changes in header and footer with some transition in icons and menus.

**Comment by mjashanks** [comment]
@AshBardhan great ☺

**Comment by mjashanks** [comment]
@AshBardhan if you are looking for something else to do, you could have a look at getting pagination to work :) 

https://jekyllrb.com/docs/pagination

also, bear in mind that any header and footer changes will apply to the entire site, and across all device sizes. Thanks!

**Comment by HishamMubarak** [comment]
Hi, is the website going to be continued in Jekyll? Wouldn't it be better if we move to any modern frameworks like Angular or React or Vue.js? Yes, It requires hosting a back-end hosted in a server, but it is scalable and I guess can attract more contributors too. Also with a good UI developed overtime, it will be easier to browse through, forms to add blog posts and more features can be added!

PS : Hosting can be removed if we use serverless architecture.
